minard/cssProc.py

```diff
@@ -0,0 +1,117 @@
+import contextlib
+import shutil
+import tempfile
+import argparse
+import sys
+import os
+import os.path
+import time
+import re
+import string
+import subprocess
+import datetime
+import json
+
+import psycopg2
+import psycopg2.extras
+import psycopg2.extensions
+
+from .db import engine_nl
+from .views import app
+
+
+RUN_TYPES = {
+    0:"Maintenance",
+    1:"Transition",
+    2:"Physics",
+    3:"Deployed Source",
+    4:"External Source",
+    5:"ECA",
+    6:"Diagnostic",
+    7:"Experimental",
+    8:"Supernova",
+    }
+
+def getStandardRunList():
+    try:
+        os.environ['PGOPTIONS'] = '-c statement_timeout=10000' #in ms
+        conn = psycopg2.connect('dbname=%s user=%s host=%s connect_timeout=5 password=%s '
+                                % ("nearline", "nearline","165.227.233.104", app.config['DB_PASS']))
+        conn.set_isolation_level(psycopg2.extensions.ISOLATION_LEVEL_AUTOCOMMIT)
+        os.environ['PGOPTIONS'] = ''
+    except Exception as e:
+        raise e
+    cursor = conn.cursor()
+
+    # conn = engine_nl.connect()
+    # result = conn.execute("SELECT run_number FROM cssproc WHERE standard_run = 't'")
+
+    cursor.execute("SELECT run_number FROM cssproc WHERE standard_run = 't' ORDER BY run_number DESC")
+    info=cursor.fetchall()
+    rtnList =[]
+    for i in info:
+        rtnList.append(i[0])
+    print rtnList
+    return rtnList
+
+def pullDownTestScores(run_number):
```

Comment:
nitpick, but I'd like to keep the style similar throughout minard. Could you change the function names to use underscores like `pull_down_test_scores`?